packages/dolmen/dolmen.0.8.1/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+maintainer: "Guillaume Bury <guillaume.bury@gmail.com>"
+authors: "Guillaume Bury <guillaume.bury@gmail.com>"
+license: "BSD-2-Clause"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
```

Comment:
Any reason not to allow the installation to run the tests of the package?

```suggestion
  ["dune" "build" "-p" name "-j" jobs "@runtest" {with-test}]
```

Reply:
there's no reason, just that I wrote the file by hand some time ago, and didn't update it in a while. I'll add the test part (and probably also the doc part I suppose).